lib/Raven/Legacy.php

```diff
@@ -14,7 +16,7 @@ class Raven_Client extends \Raven\Client
 {
 }
 
-/**
+/** @noinspection PhpDeprecationInspection
```

Comment:
My IDEA warns me about extending deprecated class \\Raven\\Compat. I suppress this warning. That's all.

This is the only deprecated class in Raven namespace.